cips/cip-reduce-issuance.md

```diff
@@ -0,0 +1,125 @@
+
+## Parameters
+
+The following parameter changes are proposed for the Celestia network:
+
+| Parameter | Current value | Proposed value | Description | Changeable via Governance |
+|-----------|---------------|----------------|-------------|---------------------------|
+| `InitialInflationRate` | 5.36% | 2.67% | The initial inflation rate at genesis (the chain has been running for 1 year) | No |
+| `MinCommissionRate` | 5% | 10% | The minimum commission rate validators can set | Yes |
+
+### APR Impact Analysis
+
+The following tables show the impact of the proposed changes on APR for different bonding ratios:
+
+**Current Schedule (5% inflation, 5% minimum commission):**
+
+| Bonding Ratio | Total Staking APR | Delegator APR | Validator APR|
+|---------------|-------------------|-----------------------|----------------------------|
+| 35% | 14.29% | 13.58% | 0.71% |
+| 50% | 10.00% | 9.50% | 0.50% |
+| 55% | 9.09% | 8.64% | 0.45% |
+| 65% | 7.69% | 7.31% | 0.38% |
+
+**Proposed Schedule (2.5% inflation, 10% minimum commission):**
+
+| Bonding Ratio | Total Staking APR | Delegator APR | Validator APR |
+|---------------|----------------|-----------------------|-------------------------|
+| 35% | 7.14% | 6.43% | 0.71% |
+| 50% | 5.00% | 4.50% | 0.50% |
+| 55% | 4.55% | 4.10% | 0.45% |
+| 65% | 3.85% | 3.46% | 0.38% |
+
+**Note**: The Validator APR is based of the total amount staked to that validator
+
+**Comparison of Changes:**
+
+| Metric | Current (5% inflation) | Proposed (2.5% inflation) | Change |
+|--------|------------------------|-------------------------|--------|
+| Total Staking APR (55% bonding) | 9.09% | 4.55% | -50% |
+| Delegator APR (55% bonding) | 8.64% | 4.10% | -53% |
+| Validator APR (55% bonding) | 0.45% | 0.45% | 0% |
+
+### Inflation Schedule Over Time
+
+The following table shows the inflation rate progression over time, starting from the current 5% rate, transitioning to 2.5% with this CIP, and continuing with the 6.7% annual disinflation rate until reaching the 1.5% target:
+
+| Year | Current Inflation (5%) | Proposed Inflation (2.5%) | Notes |
+|------|------------------------|-------------------------|-------|
+| **0** | 8.00% | 8.00% | Genesis year, no change |
+| **1** | 7.20% | 7.20% | |
+| **v4** | 5.00% | 5.00% | After CIP-29 reduction |
+| **v5** | 4.67% | 2.50% | **Proposed CIP** |
+| **2** | 4.35% | 2.33% | Regular annual disinflation applied (6.7%) |
+| **3** | 4.06% | 2.17% | Regular annual disinflation applied (6.7%) |
+| **4** | 3.79% | 2.02% | Regular annual disinflation applied (6.7%) |
+| **5** | 3.54% | 1.88% | Regular annual disinflation applied (6.7%) |
+| **6** | 3.30% | 1.75% | Regular annual disinflation applied (6.7%) |
+| **7** | 3.08% | 1.63% | Regular annual disinflation applied (6.7%) |
+| **8** | 2.87% | 1.52% | Regular annual disinflation applied (6.7%) |
+| **9** | 2.68% | 1.50% | Target inflation reached |
```

Comment:
[optional] CIP-29 has this table: https://github.com/celestiaorg/CIPs/blob/ab63f71b4a2ef8477ea1be1695e1df585768b11e/cips/cip-029.md?plain=1#L45-L66 

1. Can the table in this PR expand to 18 or 20 years?
2. [optional] is it worth expanding the decimal places so that the current inflation in this table matches the proposed inflation from the previous table?

Reply:
I think it should extend, too. 20 seems safe.

Reply:
I think extending decimals also makes sense

Reply:
But nothing else happens after year 9. There's no new information to convey when target inflation is reached